solo/client.py

```diff
@@ -409,3 +409,7 @@ def isCorrectVersion(current, target):
             self.verify_flash(sig)
 
         return sig
+
+    def has_pin (self):
```

Comment:
extra space

Reply:
what extra space? when I have spaceds at the end of the line the PR quality checker says bad.